config/browserstack/wdio.android.bs.app.conf.ts

```diff
@@ -1,4 +1,5 @@
-const { config } = require('../wdio.shared.conf');
+// @ts-ignore
+const { config }:{config: WebdriverIO.Config} = require('../wdio.shared.conf');
```

Comment:
We can use `import` here too
```suggestion
import { config } from '../wdio.shared.conf':
```

Reply:
Updated them all, thans

Reply:
still see `require` in this file

Reply:
sorry, missed that one